Dockerfile

```diff
@@ -1,15 +1,9 @@
-FROM golang:1.5
+FROM    scratch
 
-COPY . /go/src/github.com/docker/swarm
-WORKDIR /go/src/github.com/docker/swarm
+COPY    dist/bin/swarm /swarm
```

Comment:
The old Dockerfile had the same issue, right? 

I'm not sure what to do with this Dockerfile. We could leave it as an example that lets developers run it locally (but document that it doesn't have certs).